phase1/master.cfg

```
@@ -1335,6 +1336,26 @@ def prepareFactory(target):
         )
     )
 
+    factory.addStep(
+        ShellCommand(
+            name="sha256sumsclean",
+            description="Removing kmods from sha256sums",
+            descriptionDone="Kmods removed",
+            command=[
+                "sed",
+                "-i",
+                "\# *packages/kmod-.*$#d",
```

```suggestion
               r "\# *packages/kmod-.*$#d",
```

I've just double checked this and indeed, it looks to me, that this looks like a good approach, deployed with this change.